.circleci/config.yml

```diff
@@ -326,6 +326,27 @@ commands:
           environment:
             GKE_CLUSTER_NAME: << parameters.cluster >>
 
+  restore_cache_tox:
+    steps:
+      - restore_cache:
+          name: Restore .tox cache
+          keys:
+            # Note that virtualenvs are not portable! We include the job name
+            # in the key in the hopes that it encapsulates the Python version.
+            # We include the architecture (OS + processor) as well to be safe.
+            - &tox-cache v1-tox-{{ .Environment.CIRCLE_JOB }}-{{ arch }}-{{
+                checksum "requirements_dev.txt" }}-{{
```

Comment:
is there a way to invalidate cache like once a day or so? so that we don't miss latest releases of our deps?

Reply:
done, the cache key now includes the current date

Reply:
We don't change our core dependencies as often, but shouldn't this also checksum `pyproject.toml`? Or maybe some output of `pip-compile` so that it doesn't invalidate with every non-requirements change.

Reply:
hmm yeah that's a good point. I'll send a follow-up.